src/Views/FirmwareView.vala

```diff
@@ -180,6 +197,25 @@ public class About.FirmwareView : Gtk.Stack {
         update_list.show_all ();
     }
 
+    private void on_update (Fwupd.Device device, Fwupd.Release release) {
```

Comment:
That's kind of confusing to have a function `on_update` and a function `update`. Maybe one or both of these should be renamed for clarity on what they do